hieradata_aws/class/integration/service_manual_publisher_db_admin.yaml

```diff
@@ -0,0 +1,25 @@
+govuk_env_sync::tasks:
+  "pull_service_manual_publisher_production_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "0"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "service-manual-publisher_production"
+    database_hostname: "service-manual_publisher-postgresql"
```

Comment:
This should be `service-manual-publisher-postgres` (based on https://docs.google.com/spreadsheets/d/1rBZeeaT9XevaRvstMjw2YsbSsoB7IqgWjpJz5oAO9nM/edit#gid=0).

Reply:
EDIT: this should actually be `service-manual-publisher-postgresql` (i.e. with a `ql` on the end). The spreadsheet has been updated too.